### ISSUE TITLE
Intallation fails on ubuntu 20.04

### ISSUE THREAD

**Issue by alexandru-calinoiu** [ElrondNetwork/elrond-sdk#58]
Got the latest up script via

```
❯ wget -O erdpy-up.py https://raw.githubusercontent.com/ElrondNetwork/elrond-sdk/master/erdpy-up.py                                                                                               08:42:03
```

Running 

```
❯ python3 erdpy-up.py 
```

One of the steps will fail with 

```
  ERROR: Command errored out with exit status 1:
   command: /home/calin/elrondsdk/erdpy-venv/bin/python3 -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-0iqmubqv/erdpy/setup.py'"'"'; __file__='"'"'/tmp/pip-install-0iqmubqv/erdpy/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' bdist_wheel -d /tmp/pip-wheel-ofh3zn3l
       cwd: /tmp/pip-install-0iqmubqv/erdpy/
  Complete output (6 lines):
  usage: setup.py [global_opts] cmd1 [cmd1_opts] [cmd2 [cmd2_opts] ...]
     or: setup.py --help [cmd1 cmd2 ...]
     or: setup.py --help-commands
     or: setup.py cmd --help
  
  error: invalid command 'bdist_wheel'
  ----------------------------------------
  ERROR: Failed building wheel for erdpy
```

The cli seems to install, I am not able to compile contracts but I think that is a different issue.

**Comment by andreibancioiu** [comment]
@alexandru-calinoiu, as a temporary workaround, can you install the missing package via `pip3 install wheel`?

Furthermore, what do you mean by `The cli seems to install`? Can you please check the output of `erdpy --version`?

For the compilation issue, we will respond in #59.

Thank you!

**Comment by alexandru-calinoiu** [comment]
❯ erdpy --version                                                                                                     16:55:50
erdpy 0.8.5

Will try manual installation now

**Comment by alexandru-calinoiu** [comment]
❯ pip3 install wheel                                                                                                  16:56:03
Requirement already satisfied: wheel in /usr/lib/python3/dist-packages (0.34.2)

Looks like it's installed.

**Comment by tebayoso** [comment]
I think this might be fixed in #136 can you please try again?

**Comment by andreibancioiu** [comment]
Closing the issue before archiving the repository - please refer to the new repositories, instead.
